basic_module/account/src/lib.rs

```diff
@@ -0,0 +1,262 @@
+// This function will be removed when the structure of the context and conversion logic is completed.
+impl From<Vec<u8>> for Account {
+    fn from(_vec: Vec<u8>) -> Account {
+        Account {
+            sequence: 0,
+            balance: 0,
+        }
+    }
+}
+
+// Temporary measures for build.
+// This function will be removed when the structure of the context and conversion logic is completed.
+impl Account {
+    pub fn to_vec(&self) -> Vec<u8> {
+        vec![0]
+    }
+}
+
+#[derive(Clone, Copy, Default, Debug, Deserialize, Serialize, Eq)]
+pub struct Address(H160);
+
+impl Address {
+    pub fn random() -> Self {
+        Address(H160::random())
+    }
+}
+
+impl Deref for Address {
+    type Target = H160;
+
+    fn deref(&self) -> &Self::Target {
+        &self.0
+    }
+}
+
+impl DerefMut for Address {
+    fn deref_mut(&mut self) -> &mut Self::Target {
+        &mut self.0
+    }
+}
+
+impl fmt::Display for Address {
+    fn fmt(&self, f: &mut fmt::Formatter) -> Result<(), fmt::Error> {
+        write!(f, "{}", self.0)
+    }
+}
+
+impl PartialOrd for Address {
+    fn partial_cmp(&self, m: &Address) -> Option<cmp::Ordering> {
+        self.0.partial_cmp(&m.0)
+    }
+}
+
+impl Ord for Address {
+    fn cmp(&self, m: &Address) -> cmp::Ordering {
+        self.0.cmp(&m.0)
+    }
+}
+
+impl PartialEq for Address {
+    fn eq(&self, m: &Address) -> bool {
+        self.0.eq(&m.0)
+    }
+}
+
+impl Hash for Address {
+    fn hash<H: Hasher>(&self, state: &mut H) {
+        self.0.hash(state);
+    }
+}
+
+impl Encodable for Address {
+    fn rlp_append(&self, s: &mut RlpStream) {
+        self.0.rlp_append(s);
+    }
+}
+
+impl Decodable for Address {
+    fn decode(rlp: &Rlp) -> Result<Self, DecoderError> {
+        let data = H160::decode(rlp)?;
+        Ok(Address(data))
+    }
+}
+
+impl FromStr for Address {
+    type Err = String;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        let s = remove_0x_prefix(s);
+        let a = H160::from_str(s).map_err(|_| format!("Invalid address {}", s))?;
+        Ok(Address(a))
+    }
+}
+
+impl From<u64> for Address {
+    fn from(s: u64) -> Self {
+        Address(H160::from(s))
+    }
+}
+
+impl From<[u8; 20]> for Address {
+    fn from(s: [u8; 20]) -> Self {
+        Address(H160::from(s))
+    }
+}
+
+impl From<&'static str> for Address {
+    fn from(s: &'static str) -> Self {
+        s.parse().unwrap_or_else(|_| panic!("invalid string literal for {}: '{}'", stringify!(Self), s))
+    }
+}
+
+impl From<Address> for [u8; 20] {
+    fn from(a: Address) -> Self {
+        a.0.into()
+    }
+}
+
+impl AsRef<[u8]> for Address {
+    fn as_ref(&self) -> &[u8] {
+        &self.0.as_ref()
+    }
+}
+
+type Bytes = Vec<u8>;
+
+pub struct Transaction {
+    pub seq: u64,
+
+    pub fee: u64,
+
+    pub network_id: NetworkId,
+
+    pub action: Action,
+}
+
+#[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Clone, Copy, Hash)]
+pub struct NetworkId([u8; 2]);
```

Comment:
I think we should reconsider about this `NetworkId` type data. Of course module can define this kind of data but in this module I could not find the actual usage of this data. Previously, `PlatformAddress` is constructed using `Address` and `NetworkId` but removed here.

Reply:
Thank you for your good advice. I modified it.